### PR TITLE
[PF-814] missed PR fixes

### PR DESCRIPTION
**Pull request by jaycarlton**
In #401 I forgot to go back and address Zach's findings.

### CODE REVIEW

File: integration/src/main/java/scripts/testscripts/ControlledBigQueryDatasetLifecycle.java [hunk 1]
```diff
@@ -8,16 +8,9 @@
 import bio.terra.testrunner.runner.config.TestUserSpecification;
 import bio.terra.workspace.api.ControlledGcpResourceApi;
 import bio.terra.workspace.api.WorkspaceApi;
-import bio.terra.workspace.model.AccessScope;
```
Comment (jaycarlton) [inline integration/src/main/java/scripts/testscripts/ControlledBigQueryDatasetLifecycle.java:11]: drive-by organize imports (Spotless doesn't run on this module yet)

**Review by zloery — APPROVED** [review]

**Review by ddietterich — APPROVED** [review]
LGTM - thanks for following up!
